pegasus/sites.v3/code.org/public/js/volunteer-unsubscribe.js

```diff
@@ -4,13 +4,29 @@ function processVolunteerUnsubscribeResponse(data) {
 }
 
 function processVolunteerUnsubscribeError(data) {
-  $('#error-message').text('An error occurred. Please try again or contact us if you continue to receive this error.').show();
+  var errorMessage = null;
+
+  var errors = Object.keys(data.responseJSON);
+
+  for (var i = 0; i < errors.length; i++) {
+    if (errors[i] === "age_18_plus_b") {
+      errorMessage = 'An error occurred.  Please indicate whether you are at least 18, below.';
```

Comment:
To me this error isn't clear because if they are under 18 there's no way for them to continue. They may think "well I'm not yet 18, and I am trying to indicate the truth." Suggestion:

An error occurred. You must be 18 to volunteer. Please certify that you are at least 18 years old to update your preferences. If you are not yet 18, please request to remove yourself from the volunteer list by [contacting support](https://support.code.org/hc/en-us/requests/new).